### ISSUE TITLE
Check In/Out doesn't work on Vivo 3G

### ISSUE THREAD

**Issue by chgasparoto** [larruda/ciandt_tt#9]
If I try to check in/out while I'm connected in 3G the app return the message below:

Device: Motorola Moto G 1
Android: 4.4.4

![screenshot_2015-06-02-08-40-21](https://cloud.githubusercontent.com/assets/2037349/7935092/4d7b7884-0904-11e5-879b-458496126ff3.png)

**Comment by larruda** [comment]
Have you tried closing the app and trying it again? I use it everyday under Vivo 3G network and it works seamlessly. According to Google stats there are dozens of people using it with Vivo too. Also notice the clock has been synchronised, it means internet connections are working. Make sure you tap on "check" AFTER the clock is synced.

**Comment by chgasparoto** [comment]
Yes, I have tried this and clear all the data from app too. Even if I tap the button without any information the message is triggered. The clock is synchronizing like in Wi-Fi and I opened a web page to make sure it's connected to the internet.

It's weird because I asked to people here and one person has the same device and network however the OS is newer and works fine.
